ios/ExpoKit.podspec

```diff
@@ -39,7 +39,7 @@ Pod::Spec.new do |s|
     ss.dependency 'CocoaLumberjack', '~> 3.5.3'
     ss.dependency 'GoogleMaps', '~> 3.3'
     ss.dependency 'Google-Maps-iOS-Utils', '~> 2.1.0'
-    ss.dependency 'lottie-ios', '~> 3.2.3'
+    ss.dependency 'lottie-ios', '~> 3.4.0'
```

Comment:
Since lottie is already vendored by the new script, I'm almost sure we can remove this dependency from here 🤔